test/processors/image_processor_test.rb

```diff
@@ -1,13 +1,21 @@
 require 'test_helper'
 
 class ImageProcessorTest < ActiveSupport::TestCase
-  let(:processor) { ImageProcessor.new(logger: Logger.new('/dev/null')) }
+  let(:audio_monster) do
+    Minitest::Mock.new
+  end
+
+  let(:processor) do
+    ImageProcessor.new(logger: Logger.new('/dev/null')).tap do |p|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.